WebContent/WOH-index.jsp

```diff
@@ -0,0 +1,44 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE>
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>Wheel of Hunger</title>
+
+	<!-- Style Sheets -->
+	<link rel="stylesheet" type="text/css" href="site/WOH-site.css">
+	<link rel="stylesheet" type="text/css" href="index/WOH-index.css">
+
+	<!-- Latest compiled and minified CSS -->
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+	
+	<!-- jQuery library -->
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
+
+	<!-- Latest compiled JavaScript -->
+	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+</head>
+<body>
+
+	<nav class="navbar navbar-inverse">
```

Comment:
If you install the jstl library, you can put this whole bootstrap banner into an html file and then you can use a <c:include> to deploy it on the page using only one line.  This is important because then the banner will be in just one file so its very easy to edit the code.